.github/workflows/cypress-basic.yml

```diff
@@ -0,0 +1,114 @@
+# This is a basic workflow that is manually triggered
+
+name: Cypress automation test - [basic]
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      wazuh-version:
+        description: 'Wazuh Version'
+        default: 'v4.3.4'
+        required: true
+      elastic-version: 
+        description: 'Elastic Version'
+        default: '7.16.3'
+        required: true
+      ubuntu-cypress-version: 
+        description: 'Ubuntu Cypress Version'
+        default: '3.0.0'
+        required: true
+      url-package:
+        description: 'Package Download Url'
+        required: false 
+        default: 'https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuh_kibana-4.3.4_7.16.3-1.zip'       
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  setup-wazuh-kibana-app:
+    name: Run setup environment wazuh kibana app
+    runs-on: ubuntu-18.04
+    steps:
+     - name: Step 01 - Set up environment variables
+       run: |
+        echo "PATH_TEMPLATE=$GITHUB_WORKSPACE/wazuh-app-environments/templates_elastic_prod/es_basic-wz_cluster-agent/" >> $GITHUB_ENV
+
+        echo "Information about the workflow (basic):
+        wazuh-version: ${{ github.event.inputs.wazuh-version }}
+        elastic-version: ${{ github.event.inputs.elastic-version }}
+        ubuntu-cypress-version: ${{ github.event.inputs.ubuntu-cypress-version }}
+        url-package: ${{ github.event.inputs.url-package }}"
+        
+     - name: Step 02 - Download Project wazuh-app-environments 
+       uses: actions/checkout@v2
+       with:
+        repository: matiasmoreno876/wazuh-app-environments
```

Comment:
**thought**: we should have the environments in this repository.

Reply:
Yes, for the time being we will continue to use them in this way until the new environments are integrated into the project.